uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/main/java/org/uberfire/ext/metadata/io/IOServiceIndexedImpl.java

```diff
@@ -364,8 +363,11 @@ private void queueEvent(final Set<Path> eventRealPaths, WatchEvent event,
                         }
 
                         private void queueDeleteEvent(WatchEvent object, final WatchContext context, final IndexerDispatcher dispatcher) throws DisposedException {
-                            final Path oldPath = context.getOldPath();
-                            dispatcher.offer(new DeletedFileEvent(oldPath));
+                            Path path = context.getOldPath();
+                            if(!path.getFileName().toString().startsWith(".")){
+                                final Path oldPath = context.getOldPath();
```

Comment:
@jhrcek in this case I prefere to keep the startsWith, because this issue was very hard to find, and I don't want to lose readability at least in this. If we loose too much performance I will consider to change this to `charAt`, but for now I prefere it like that. Thanks!!